GeneralsMD/Code/Main/SDL3Main.cpp

```diff
@@ -70,14 +79,38 @@ static Bool initializeAppWindows(Bool runWindowed)
     startHeight = DEFAULT_YRESOLUTION;
   }
 
-  SDL_Window *window =
-      SDL_CreateWindow("Command and Conquer Generals", startWidth, startHeight, SDL_WINDOW_VULKAN);
-  SDL_ShowWindow(window);
+  TheSDL3Window = SDL_CreateWindow(
+      "Command and Conquer Generals", startWidth, startHeight,
+      SDL_WINDOW_VULKAN | SDL_WINDOW_RESIZABLE | SDL_WINDOW_HIDDEN);
```

Comment:
I don't think you'll need the resizable flag here but I could be wrong. I removed it in my pr to allow the in game setting to control it so the window size was always accurate